src/core/server/rendering/rendering_service.tsx

```diff
@@ -135,7 +135,7 @@ export class RenderingService {
                 defaultUrl: brandingAssignment.loadingLogoDefault,
                 darkModeUrl: brandingAssignment.loadingLogoDarkmode,
               },
-              favicon: '',
+              favicon: brandingAssignment.favicon,
```

Comment:
faviconUrl?

Reply:
Sounds good. Will make it consistent with other configs.